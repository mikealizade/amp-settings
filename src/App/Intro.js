import React, { Fragment } from 'react';

export const Intro = () => (
    <Fragment>
        <p className='intro'>Welcome to Guitarist Amp Settings. This site aims to help guitarists find their
        favourite artists' amplifier settings with minimal fuss or effort.</p>
        <p className='intro'>Please use the feedback form for any suggestions, improvements or any amp settingscddcsdcsd
            that you would like to see on the site.</p>
    </Fragment>
)