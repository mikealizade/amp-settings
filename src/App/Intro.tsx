import React from 'react';

interface Props {
  para: string,
  para2: string
}

export const Intro: React.FC<Props> = ({ para, para2 }) => (
  <>
    <p className='intro'>{ para }</p>
    <p className='intro'>{ para2 }</p>
  </>
);
