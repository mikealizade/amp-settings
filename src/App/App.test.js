import React, { Fragment } from 'react';
// import ReactDOM from 'react-dom';
// import App from './App';
// import { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Intro } from './Intro';

configure({ adapter: new Adapter() });

describe('<Intro>', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Fragment>
          <p className='intro'>Welcome to Guitarist Amp Settings. This site aims to help guitarists find their
              favourite artists' amplifier settings with minimal fuss or effort.</p>
          <p className='intro'>Please use the feedback form for any suggestions, improvements or any amp settingscddcsdcsd
                  that you would like to see on the site.</p>
        </Fragment>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
