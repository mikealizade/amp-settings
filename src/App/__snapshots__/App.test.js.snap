// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Intro> renders correctly 1`] = `
Array [
  <p
    className="intro"
  >
    Welcome to Guitarist Amp Settings. This site aims to help guitarists find their favourite artists' amplifier settings with minimal fuss or effort.
  </p>,
  <p
    className="intro"
  >
    Please use the feedback form for any suggestions, improvements or any amp settingscddcsdcsd that you would like to see on the site.
  </p>,
]
`;
